### ISSUE TITLE
Kein Upload von Dateien über CMS möglich

### ISSUE THREAD

**Issue by dangoerr** [lucascranach/cda-orga#138]
### URL of the buggy page?

https://lucascranach.org/index.php/dashboard/files/search

### Contact Details

DG

### What happened?

Hi Markus,
Concrete 5 lässt mich aktuell keine jpeg- und pdf-Dateien hochladen, die etwa 1 MB überschreiten. Fehlermeldung: 
"413 Request Entity Too Large
nginx/1.22.1"
Es sind aber diverse Dateien, die deutlich größer sind, auf dem Server. Wurde durch ein Update irgendeine Upload-Limit aktiv?

**Comment by markusalterauge** [comment]
Hi @dangoerr,
wie groß soll das Limit sein, damit du ordentlich arbeiten kannst?

**Comment by markusalterauge** [comment]
Ich habe einen Befehl eingefügt, der das Limit auf 10 MB setzt. Bitte teste mal, ob das funktioniert.

**Comment by dangoerr** [comment]
Danke, aber funktioniert lieder noch nicht (gleiche Fehlermeldung), trotz Neuanmeldung im System. In Kürze wollen wir ein neues Video online stellen, das knapp 800 MB hat. Also dreh das Limit bitte auf 900 MB oder so.

**Comment by markusalterauge** [comment]
Erledigt. Gern nochmal probieren.

**Comment by dangoerr** [comment]
Sorry, weiterhin die gleiche Fehlermeldung...

**Comment by markusalterauge** [comment]
Hallo Daniel, ist definitiv noch das "nginx/1.22.1" in der Fehlermeldung enthalten?

**Comment by dangoerr** [comment]
Ja, ist es

**Comment by markusalterauge** [comment]
@dangoerr, probier es bitte noch einmal. Habe die Uploadgröße jetzt auf 899,99 MB festgelegt.

**Comment by dangoerr** [comment]
Weiterhin ohne Erfolg und weiterhin mit "nginx/1.22.1"-Fehler. 
:-/

**Comment by markusalterauge** [comment]
Ich habe jetzt mal auf 2GB hochgeschraubt. Lädst du das denn unter Verwaltung -> Dateien hoch? Oder wie machst du das?

**Comment by dangoerr** [comment]
Ja, hier mache ich das immer:
![grafik](https://github.com/lucascranach/cda-orga/assets/91158095/a0761337-09f7-45e7-a402-64495e1d9744)
Ich teste immer mit einer 3,7 MB pdf und einem 1,8 MB jpeg. Führt wie gesagt immer noch zur gleichen Fehlermeldung.

**Comment by markusalterauge** [comment]
Am besten noch mal die Webseite neu laden.
Concrete 5 setzt momentan das Limit von 256MB. 
Ich weiß noch nicht ob ich daran rumschrauben darf.

**Comment by cnoss** [comment]
Hi, also da müssen wir mal sprechen … ein 800MB Film wird für das C5 zu groß sein und ich weiß auch nicht ob der Server das gescheit streamt … @dangoerr bitte mal am Mittwoch in den Jourfix kommen, dann können wir uns das mal zusammen anschauen … habt ihr schon mal so große Daten ins C5 gepackt?

**Comment by dangoerr** [comment]
Ja, da hast Du recht. Die größte Videodatei bisher ist 171 MB. Da werden wir das neue Video sicher kleinerrechnen müssen, sofern das geht. Bin gerade in Berlin und morgen in Wittenberg mit vollem Programm und kann daher nicht am JF teilnehmen. Ab Freitag wäre ich bei Bedarf wieder telefonisch erreichbar.

**Comment by yannic-bruegger** [comment]
Soll ich das issue schließen? Bzw. haben wir eine Lösung gefunden? @dangoerr

**Comment by dangoerr** [comment]
Leider noch nicht, der Fehler tritt weiterin auf und verhindert, dass ich neue Dateien (auch kleinere) hochladen kann.

**Comment by dangoerr** [comment]
Habe heute noch einmal mit Frau Bolz Rücksprache gehalten, die die Website erstellt hat. Sie konnte ausschließen, dass es sich um ein Website/CMS-Problem handelt und vermutet die falschen Parameter der nginx-Datei, die wohl serverseitig angepasst wird. 
Hattet ihr das schon gecheckt?
Diese Lösungansätze schickte sie mir:
https://stackoverflow.com/questions/24306335/413-request-entity-too-large-file-upload-issue

**Comment by cnoss** [comment]
@dangoerr @CranacharchivGunnar ich habe hier nun den Server anders konfiguriert und Herr Solleder hat die Firewall angepasst, so dass jetzt Bilder/ Dateien bis 20MB hochgeladen werden können – bitte mal austesten und bei Erfolg Ticket hier schließen.
